lib/api/assertions/titleEquals.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * The .assert.title() command, which will become an alias of this and will be deprecated
+ * 
```

Comment:
need to update the docs

Reply:
I didn't understand, can you please elaborate